imap_processing/cdf/config/imap_codice_l1a_variable_attrs.yaml

```diff
@@ -25,94 +25,83 @@ codice_support_attrs: &support_default
     VAR_TYPE: support_data
 
 # <=== Coordinates ===>
-energy_attrs:
+esa_step:
     <<: *default
     CATDESC: Energy per charge (E/q) sweeping step
-    FIELDNAM: Energy step
+    FIELDNAM: Energy Index
     FORMAT: I3
-    LABLAXIS: energy step
+    LABLAXIS: Energy Index
+    SI_CONVERSION: " > "
     UNITS: ' '
     VALIDMIN: 0
-    VALIDMAX: 127
+    VALIDMAX: 128
```

Comment:
Are you sure you want this to be 128? Just double checking

Reply:
I pulled this from the L1A validation CDFs that joey gave me but I do think it should be 127. I will double check with Joey.

Reply:
Joey confirmed that this should be `127` (and that the exact values in his validation CDFs should not be considered the strict requirement).